### PR TITLE
Feature/new password edit form

### PR DESCRIPTION
**Pull request by Alisa1989**
The object of this pull request was to create a new form to change the user password.

The initial objective was actually to make the existing settings form functional but it didn't make sense how it was set, as it prompted the user to change the password and email at the same time without actually prompting for a new password.

I consulted my project leader and we came to the conclusion that the settings page would need to be split into two separate pages, one for changing the email, one for changing the password.

In this PR I created and styled the new parent/changepassword page

[trello link](https://trello.com/c/tRqKT6Bi/687-make-edit-parent-settings-functional)

https://user-images.githubusercontent.com/37967343/140441700-4456e9c4-ba45-483c-8e67-c111699c934c.mp4

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -241,6 +242,11 @@ function App() {
             <ParentSettings LoadingComponent={ParentLoadingComponent} />
           )}
         />
+        <Route
+          exact
+          path="/parent/changepassword"
+          component={() => <ChangePassword />}
+        />
```
Comment (Fry-Fr) [inline src/index.js:255]: Should the component be rendered by ProtectedRoute instead? Right now you have Route to render ChangePassword.

**Review by artofmayhem — DISMISSED** [review]
Well conceived and executed change.

File: src/components/pages/FamilySettings/ChangePassword.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import ParentNavTopBar from '../../common/ParentNavTopBar';
+import { Form, Input, Button, Row, Col, Layout, Card } from 'antd';
+
```
Comment (just-kato) [inline src/components/pages/FamilySettings/ChangePassword.js:4]: I know we talked about you creating this form. It's nice to see it come to life. Great job

File: src/index.js [hunk 3]
```diff
@@ -44,6 +44,7 @@ import { ParentContact } from './components/pages/ParentContact';
 import { NewParentDashboard } from './components/pages/NewParentDashboard';
 import { SupportPage } from './components/pages/SupportPage';
 import { ParentSettings } from './components/pages/FamilySettings';
+import ChangePassword from './components/pages/FamilySettings/ChangePassword';
```
Comment (just-kato) [inline src/index.js:47]: Nice work

File: src/index.js [hunk 4]
```diff
@@ -241,6 +242,11 @@ function App() {
             <ParentSettings LoadingComponent={ParentLoadingComponent} />
           )}
         />
+        <Route
+          exact
```
Comment (just-kato) [inline src/index.js:252]: Great job

File: src/styles/less/ChangePassword.less [hunk 5]
```diff
@@ -0,0 +1,3 @@
+.form-input{
+    border-radius: 0px;
```
Comment (just-kato) [inline src/styles/less/ChangePassword.less:2]: Nice work

File: src/styles/less/index.less [hunk 6]
```diff
@@ -41,6 +41,7 @@
 @import 'NewChildCard';
 @import 'GamemodeStyles';
 @import 'AudioBook';
+@import 'ChangePassword';
```
Comment (just-kato) [inline src/styles/less/index.less:44]: Great job

**Review by just-kato — DISMISSED** [review]

File: src/styles/less/index.less (hunk 6 above)
Comment (BryanPAnderson) [inline src/styles/less/index.less:44]: great work

**Review by BryanPAnderson — COMMENTED** [review]
good job!

File: src/index.js [hunk 7]
```diff
@@ -242,6 +243,11 @@ function App() {
...
+          path="/parent/changepassword"
+          component={() => <ChangePassword />}
+        />
         {/* UNDER CONSTRUCTION HOLDER / New path to point to /gameplay/mission/next-steps NOT for DELETE */}
```
Comment (tinomen) [inline src/index.js:256]: Is this comment needed? If we don't know, then delete it.

**Review by tinomen — CHANGES_REQUESTED** [review]
Lets get some of these suggested changes taken care of and merge.

**Review by Fry-Fr — DISMISSED** [review]
changes appear to be resolved. this works fine.

**Review by StnsGeneral — DISMISSED** [review]
Looks much better after the changes.

**Pull request closed without merging**